node/middlewares/notify.ts

```diff
@@ -45,6 +45,25 @@ const replaceIfChanged = async <T>(
   return false
 }
 
+const getAllCategories = async (categoryId: string | undefined, ctx: Context): Promise<IdentifiedCategory[]> => {
+  const { clients: { catalogGraphQL }, vtex: { logger }} = ctx
+  if (!categoryId) {
+    return []
+  }
+  const categoryResponse = await catalogGraphQL.category(categoryId).catch(logError(logger))
```

Comment:
This `.catch(logError(logger))` may make us loose indexing a category due to a timeout. I'd rather remove this and error to courrier so it retries